gitgud/skills/user_messages.py

```diff
@@ -219,13 +219,21 @@ def content_display_data(data):
     for branch_name in tree['branches']:
         target = tree['branches'][branch_name]['target']
         if target not in referred_by:
-            referred_by.update({target:[branch_name]})
+            referred_by.update({target: [branch_name]})
         else:
             referred_by[target].append(branch_name)
     for target in referred_by:
         referred_by[target] = ", ".join(referred_by[target])
 
     # Iterate through heads to get a consistent output.
+    if working:
+        print("Working Directory:")
+        working_data = file_operator.get_working_directory_content()
+        content_display_data(working_data)
+    if staging:
+        print("Staging Directory:")
+        staging_data = file_operator.get_staging_content()
+        content_display_data(staging_data)
```

Comment:
This part seems isolated, so maybe they can go in their own function?

Reply:
See my [previous comment](https://github.com/benthayer/git-gud/pull/310#discussion_r485926835).